tutorials-v5/time-evolution/013_nonmarkovian_monte_carlo.md

```diff
@@ -0,0 +1,563 @@
+    alpha=0.2, color='C0'
+)
+
+plt.plot(times2, MESol2.expect[1], color='C1',
+         label=r"$\langle e_1 \mid \rho \mid e_1 \rangle$")
+plt.plot(times2, MCSol2.average_expect[1], 'x', color='C1')
+plt.fill_between(
+    times2,
+    MCSol2.average_expect[1] - MCSol2.std_expect[1] / np.sqrt(ntraj),
+    MCSol2.average_expect[1] + MCSol2.std_expect[1] / np.sqrt(ntraj),
+    alpha=0.2, color='C1'
+)
+
+plt.plot(times2, MESol2.expect[2], color='C2',
+         label=r"$\langle e_2 \mid \rho \mid e_2 \rangle$")
+plt.plot(times2, MCSol2.average_expect[2], 'x', color='C2')
+plt.fill_between(
+    times2,
+    MCSol2.average_expect[2] - MCSol2.std_expect[2] / np.sqrt(ntraj),
+    MCSol2.average_expect[2] + MCSol2.std_expect[2] / np.sqrt(ntraj),
+    alpha=0.2, color='C2'
+)
+
+plt.plot(times2, np.ones_like(times2), color="0.5",
+         label=r"$\operatorname{tr} \rho$")
+plt.plot(times2, MCSol2.average_trace, "x", color="0.5")
+plt.fill_between(
+    times2,
+    MCSol2.average_trace - MCSol2.std_trace / np.sqrt(ntraj),
+    MCSol2.average_trace + MCSol2.std_trace / np.sqrt(ntraj),
+    alpha=0.2, color="0.5"
+)
+
+plt.xlabel(r"$t$")
+plt.ylabel('Expectation values')
+plt.legend()
+
+plt.show()
+```
+
+When computing a quantity such as the expectation value of $H$ with `nm_mcsolve`, the solver first computes that quantity along each trajectory and then averages over the trajectory values, weighing each trajectory with the value of the associated influence martingale $\mu$. In the plot below, we show the evolution of this expectation value along a number of individual trajectories. (The plots do not include the martingale weight; therefore, the vertical axis is labelled as $\langle H \rangle\, /\, \mu$.) The shading of the lines in this plot represent the value of the martingale. The solid red line is the exact value of $\langle H \rangle$, which would be obtained by averaging $( \langle H \rangle\, /\, \mu )$ with weights $\mu$ over sufficiently many trajectories.
+
+The plot shows that the number of trajectories contributing to the average declines rapidly, due to two effects. First, jumps with the jump operators $L_1$ or $L_2$ bring the system into the ground state. A trajectory that enters the ground state at any time cannot leave it again. Second, the solver adds a third jump channel with zero jump rate. A jump in this channel sets the martingale of the corresponding trajectory to zero for the remainder of the time. In the plot below, we visualize this effect by making the trajectories stop before the end of the time interval.
+
+If $\langle H \rangle$ went to zero, a declining number of contributing trajectories would not be an issue. However, due to the negative rates, the ensemble average does not converge to the steady state $|0\rangle\langle 0|$. Therefore, the weights of the remaining contributing trajectories must grow rapidly to counteract the non-contributing trajectories, leading to a rapidly growing error.
+
```

Comment:
Just a small suggested text change here:

"""
If $\langle H \rangle$ went to zero, a declining number of contributing trajectories would not be an issue. However, due to the negative rates, the ensemble average does not converge to $|0\rangle\langle 0|$. Therefore, the weights of the remaining contributing trajectories (as indicated by their shading) must grow rapidly to counteract the non-contributing trajectories, leading to a rapidly growing error because of insufficient sampling of such trajectories.
"""